plugins/client/README.md

```diff
@@ -1,11 +1,70 @@
-# client
+# @leancodepl/contractsgenerator-typescript-plugin-client
```

Comment:
What do you think about putting short sample of the generated output (and maybe input that was used) for each plugin? Each one has only a single short sentence explaining the purpose of the plugin, and I believe it would be much easier to understand it, if one knew what the plugin is going to produce.

Reply:
Done